### ISSUE TITLE
Can you replicate the result of the original paper？

### ISSUE THREAD

**Issue by Bryan-bai** [fshamshirdar/DeepVO#1]
Hello,
I cannot replicate the result of the original paper. If you can do it, please upload the pretrained weights. Thanks!
Because I trained it for 100 epochs, the network almost learned nothing. The testing loss is too high and the results are not visually convincing at all. Have you found a way to solve it? Thanks!

**Comment by chenmy17** [comment]
Hello,
I have met the same problem. The results are discrete and the net had learned nothing. Could you plz give some advice.
Thanks.

**Comment by Bryan-bai** [comment]
Hi，I try dropout and batchnorm，and still cannot get a satisfactory result. 
Because the original paper does not give the training detail, I doubt the results facticity.
You can try other method and if you have a good result, plz tell me.
Thanks.

**Comment by fshamshirdar** [comment]
Hi, I was not planning to replicate the original paper at the first place, I shouldn't have named this repo DeepVO, sorry for the misleading . I was doing some experiments in different branches, such as concatenating PlaceNet and DeepVO network which I stopped at some point. Current implementation in DeepVO branch is not quite the same as the paper, LSTMCell size should be 1000, image size is 1241x376x3, network should be trained by different trajectory lengths to have a better generalization, data augmentation, and I guess current Loss function is different and experimental too. By the way, I got some good results trying different methods, but unfortunately I don't have any checkpoint :(.

**Comment by Bryan-bai** [comment]
Hi，thank you for your advice.
I modified this repo according to the orignal paper. I also trained the network by different trajectory lengths. The loss function is the same with the papaer, lr=1e-4. However, the results are not visually convincing  at all.  Can you give me your methods and the training detail？Thanks.

**Comment by fshamshirdar** [comment]
I am sorry, I can't quite recall since it is been a year, but I am pretty sure that I've got convincing results on validation set, not on a different dataset. Are you testing it on a different dataset?

**Comment by Bryan-bai** [comment]
No, the validation set is Kitti 03 04 05 06 07 10 and training set is 00 02 08 09. In validation set, the results is terrible. 
Thanks all the same.

**Comment by fshamshirdar** [comment]
Apart from the method and training strategy, your training set is too small.

**Comment by Bryan-bai** [comment]
It is the same with the original paper.

**Comment by fshamshirdar** [comment]
I've read the training section again. You're right. But you need to load pre-trained FlowNet weights to the current CNN model.
